CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.25) # ubuntu 23.04 Fedora 36
 
 # Qt6.4+ only
 
-option(WITH_FFMPEG_PLAYER "Enable support for FFMPEG player" ON)
+option(WITH_FFMPEG_PLAYER "Enable support for FFMPEG player" OFF)
```

Comment:
Maybe don't change this default yet. What if someone uses older Qt. Also, Qt multimedia has multiple backends now, FFmpeg isn't guaranteed to be the default on some Linux distros.

Reply:
This cmake file doesn't include qt5, and older version before 6.5 is buggy. so I think it's doesn't matter.

Reply:
Just do that with -DWITH_FFMPEG_PLAYER=ON. I don't think it's matter, but most versions in our workflows don't need it.

Reply:
There are a few places in the wild that aren't in our control. I think we can wait some time and delete all FFmpeg code. We don't need to turn it off now.

- https://src.fedoraproject.org/rpms/goldendict-ng/blob/rawhide/f/goldendict-ng.spec#_80
- https://codeberg.org/hashirama/ajattix/src/commit/6108b9c4839a5760ae5bfb76451c945257b7419f/ajatt/packages/dictionaries.scm#L53-L54
- https://github.com/archlinuxcn/repo/blob/770b047a3591e6e24f0962e784bbbaae6acceb10/archlinuxcn/goldendict-ng-git/PKGBUILD#L53-L57 (this is in my control, but i don't want to change it :sweat_smile: )

Reply:
Maybe we should add deprecated warning after qt6.4+ are widely used.

Reply:
I don't know the plan, maybe we can cut a release for the last version that supports Qt5 first :sweat_smile:  @xiaoyifang